frontend/src/components/VariantListPage/VariantListVariants.tsx

```diff
@@ -112,12 +112,13 @@ const VariantListVariants = (props: VariantListVariantsProps) => {
     setPopulationsDisplayedInTable,
   ] = useState<GnomadPopulationId[]>([]);
   const [includeAC0Variants, setIncludeAC0Variants] = useState(false);
+  const [searchText, setSearchText] = useState("");
 
   const { variants, structural_variants } = variantList;
 
-  const renderedVariants = !structural_variants
-    ? variants
-    : combineVariants(variants, structural_variants);
+  const renderedVariants = useMemo(() => {
+    return combineVariants(variants, structural_variants);
+  }, [variants, structural_variants]);
```

Comment:
**issue**: it appears as though a ternary got dropped here, I believe this ternary is needed since very old variant lists would return a null type for SVs, in which case we'd want to not run combineVariants.

**suggestion**: I think re-working the `combineVariants` to handle null would probably be the better play, but as that's not really the point of this PR, we could instead just re-add the ternary that got dropped, and keep the `useMemo` hook if we want.